src/index.spec.ts

```diff
@@ -28,7 +29,7 @@ function dummyDataUser () {
   const providedIndex = ref(dummyDocumentIndex(['firstName', 'lastName', 'email']))
 
   for (let i = 0; i < faker.datatype.number(); i++) {
-    const user = { firstName: faker.name.firstName(), lastName: faker.name.lastName(), id: i }
+    const user = { firstName: faker.phone.number(), lastName: faker.phone.imei(), id: i }
```

Comment:
Using `faker.phone` for _firstName_ and _lastName_ is not ok.
Changes the object structure to `{ fullName: faker.name.fullName, email: faker.internet.email(), phone: faker.phone.number(), id: i }`